### PR TITLE
Addition of a kubernetes deployment example and documentation in README.md

### PR DESCRIPTION
**Pull request by kaspernissen**
This PR adds a small example of how you can run the exporter in kubernetes. Further it adds instructions on how to use it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,6 +95,41 @@ time="2019-01-10T19:57:01Z" level=info msg="Listening on :9532" source="main.go:
...
+  - source_labels: [__address__, __meta_kubernetes_pod_annotation_prometheus_io_port]
+    action: replace
+    regex: (.+):(?:\d+);(\d+)
+    replacement: ${1}:${2}
+    target_label: __address__
+  - action: labelmap
+    regex: __meta_kubernetes_pod_label_(.+)
+```
+
+To deploy it to your kubernetes cluster run the following command:
```
Comment (Crevil) [inline README.md]: `command` should be plural -> `commands`

File: examples/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      labels:
+        app: snyk-exporter
+      annotations:
+        prometheus.io/scrape: 'true'
+        prometheus.io/port: '9532'
+    spec:
+      containers:
+      - name: snyk-exporter
+        image: quay.io/lunarway/snyk_exporter:v1.0.0
+        args: ["--snyk.api-token", "$(SNYK_API_TOKEN)", "--log.level", "debug"]
```
Comment (Crevil) [inline examples/deployment.yaml:23]: Maybe we don't need the debug logging in the example? Or maybe it is actually nice to see what it does when testing it out? Hmm...

File: README.md [hunk 3]
```diff
@@ -95,6 +95,41 @@ time="2019-01-10T19:57:01Z" level=info msg="Listening on :9532" source="main.go:
 time="2019-01-10T19:57:01Z" level=info msg="Running Snyk API scraper..." source="main.go:94"
 ```
 
+# Deployment
+
+To deploy the exporter in Kubernetes, you can find a simple Kubernetes deployment and secret yaml in the `examples` folder. You have to add your snyk token in the `secrets.yaml` and/or the snyk organizations that you want to get metrics from in the args section of the `deployment.yaml`. If you don't specify a snyk-organization, the exporter will scrape all organizations the key provides access to. The examples assumes that you have a namespace in kubernetes named: `monitoring`.
```
Comment (Crevil) [inline README.md]: Can you change `key` to `token` just to make it super clear that it's the Snyk API token?

**Review by Crevil — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kaspernissen) [reply README.md]: 👍

File: README.md (hunk 3 above)
Comment (kaspernissen) [reply README.md]: 👍

File: examples/deployment.yaml (hunk 2 above)
Comment (kaspernissen) [reply examples/deployment.yaml:23]: Let's keep it as is

**Review by Crevil — APPROVED** [review]

**Comment by kaspernissen** [comment]
the minor fixes should be fixed now.

**Pull request merged**